config/docker/clang-base/Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
    libhugetlbfs-dev:arm64 \
    libpopt-dev:arm64 \
    libasound2-dev:arm64 \
-   pkg-config:arm64
+   pkg-config
```

Comment:
I guess we don't strictly need to list `pkg-config` multiple times. Is this more to keep the list self-contained in case we want to split the Dockerfile into per-arch fragments? In this case, maybe `libasound2-dev` should be listed again as well?

Reply:
I'm not sure of the rationale TBH. As things stand we should be able to just merge everything.